controller/array_action/array_mediator_ds8k.py

```diff
@@ -469,31 +463,38 @@ def _get_flashcopy_process(self, flashcopy_id, not_exist_err=True):
             logger.exception(ex)
             raise ex
 
+    def is_snapshot(self, api_snapshot):
```

Comment:
consider renaming `api_snapshot` to `api_volume`

Reply:
done